srcalc/src/main/java/gov/va/med/srcalc/domain/Calculation.java

```diff
@@ -225,18 +225,57 @@ public String buildNoteBody()
     	// Build the note body where each section is separated by a blank line
 		// Specialty
     	returnString.append(String.format("Specialty = %s%n%n", this.getSpecialty().toString()));
+    	//Procedures
+    	for(final Value value: this.getProcedureValues())
+    	{
+    		returnString.append(String.format("%s = %s%n", value.getVariable().getDisplayName(), value.getDisplayString()));
+    	}
+    	// Model results
+		returnString.append(String.format("%nResults%n%n"));
+		for(final String key: this.getOutcomes().keySet())
+		{
+			returnString.append(String.format("%s = %.1f%%%n", key, this.getOutcomes().get(key) * 100.0));
+		}
 		// Variable display names and values
     	returnString.append(String.format("Calculation Inputs%n"));
-		for(final Value value: this.getValues())
+		for(final Value value: this.getNonProcedureValues())
 		{
 			returnString.append(String.format("%s = %s%n", value.getVariable().getDisplayName(), value.getDisplayString()));
 		}
-		// Model results
-		returnString.append(String.format("%nResults%n"));
-		for(final String key: this.getOutcomes().keySet())
-		{
-			returnString.append(String.format("%s = %.1f%%%n", key, this.getOutcomes().get(key) * 100.0));
-		}
 		return returnString.toString();
 	}
+    
+    /**
+     * Returns a set of only {@link ProcedureValue} objects
```

Comment:
The Javadoc isn't very clear. Are these just _any_ ProcedureValue objects? Identify that it's the input values for the calculation.